etna/analysis/plotters.py

```diff
@@ -37,24 +38,93 @@ def prepare_axes(segments: List[str], columns_num: int, figsize: Tuple[int, int]
     return ax
 
 
+def _get_existing_quantiles(ts: "TSDataset") -> Set[float]:
+    """Get quantiles that are present inside the TSDataset."""
+    cols = [col for col in ts.columns.get_level_values("feature").unique().tolist() if col.startswith("target_0.")]
+    existing_quantiles = {float(col[len("target_") :]) for col in cols}
+    return existing_quantiles
+
+
+def _select_quantiles(forecast_results: Dict[str, "TSDataset"], quantiles: Optional[List[float]]) -> List[float]:
+    """Select quantiles from the forecast results."""
+    # restrict to use only two quantiles, in other case the plot will be messy
```

Comment:
do we really need it?

as a user I would wait `plot_...` to plot all the data from ts without any filters

Reply:
Look at the last plots here: https://github.com/tinkoff-ai/etna/pull/538 (not the first ones)
Do you think it will be good to plot something like this for all the forecasts?

Should we in this case select quantiles, that are common for all the forecasts or some other strategy should be applied?